src/dataviews/dataview-model-base.js

```diff
@@ -71,21 +78,18 @@ module.exports = Model.extend({
   initialize: function (attrs, opts) {
     attrs = attrs || {};
     opts = opts || {};
+    util.checkRequiredOpts(opts, REQUIRED_OPTS, 'DataviewModelBase');
+
+    this._map = opts.map;
+    this._vis = opts.vis;
```

Comment:
@ivanmalagon don't you think this makes it easier to know what are the "private" instance variables of this class (plus it only declares the variables we explicitly "extract" from `opts`?

Reply:
In this case, you get used to it.

What I find worse is the lack of knowing what the attributes are. We don't tend to set the `attributes` property in Backbone classes and then knowing the different attributes is a mistery. But this is thing of how we use Backbone through all repos (and it has nothing to to with this PR).